client/my-sites/themes/multi-site.jsx

```diff
@@ -48,6 +48,10 @@ const ThemesMultiSite = React.createClass( {
 
 	getButtonOptions() {
 		const buttonOptions = {
+			customize: {
+				action: theme => this.showSiteSelectorModal( 'customize', theme ),
+				hideForTheme: theme => ! theme.active
```

Comment:
I don't think we need `customize` on the multisite view, because `theme.active` is never true.